Tasks/NuGetInstaller/nugetinstaller.ts

```diff
@@ -2,9 +2,11 @@
 /// <reference path="../../definitions/Q.d.ts" />
 /// <reference path="../../definitions/vsts-task-lib.d.ts" />
 
+import fs = require('fs');
 import path = require('path');
 import Q = require('q');
 import tl = require('vsts-task-lib/task');
+import helper = require('./vsonugethelper');
```

Comment:
vso-nuget-helper might be a better name similar to the pattern followed by other utils.